WordPress/src/main/res/values/attrs.xml

```diff
@@ -107,4 +107,17 @@
         <attr name="wpTextDialogSubtitle" format="string" />
     </declare-styleable>
 
+    <!--
+        WPLoginInputRow
+    -->
+    <declare-styleable name="wpLoginInputRow">
+        <attr name="wpIconDrawable" format="reference"/>
+        <attr name="wpIconContentDescription" format="string"/>
+        <attr name="android:inputType" />
```

Comment:
Hmm, the other attributes are actually inherited from Android's set of attributes and are used as normal. The `wp` prefixed ones are basically the "real" custom ones and that fact becomes clearer when using them. I'd suggest we keep the different naming. WDYT?